packages/syft/src/syft/client/client.py

```diff
@@ -201,16 +192,28 @@ def session(self) -> Session:
             self.session_cache = session
         return self.session_cache
 
-    def _make_get(self, path: str, params: dict | None = None) -> bytes:
+    def _make_get(
+        self, path: str, params: dict | None = None, stream: bool = False
+    ) -> bytes | Iterable:
         if params is None:
-            return self._make_get_no_params(path)
-        url = self.url.with_path(path)
+            return self._make_get_no_params(path, stream=stream)
+
+        url = self.url
+
+        if self.rathole_token:
```

Comment:
If rathole token is part of connection, then all GET, POST, PUT calls made via the internal proxy url with Hostname param as Node name.